rasa/core/actions/forms.py

```diff
@@ -632,7 +632,14 @@ async def do(
         events_so_far: List[Event],
     ) -> List[Event]:
         """Executes form loop after activation."""
-        events = await self._validate_if_required(tracker, domain, output_channel, nlg)
+        events: List[Event] = []
+        filtered_events = [
```

Comment:
Could you please add a multi-line comment explaining why it is required to filter the `ActiveLoop` event (which is added during form activation) + your observations of why we can skip the validation if `events_so_far` contains events other than `ActiveLoop`.